omni-chromium/author.js

```diff
@@ -7,7 +7,7 @@ function AuthorSearcher(query) {
 inherits(AuthorSearcher, Searcher);
 
 AuthorSearcher.prototype.getSuggestionsURL = function() {
-  return 'https://code.google.com/p/chromium/feeds/issueOptions';
+  return 'https://bugs.chromium.org/p/chromium/feeds/issueOptions';
```

Comment:
Yeah, pity this doesn't work. Would be nice to debug at some point, weird that exatly the same request works with wget.